src/containers/mobile/landing/hero/hero-component.jsx

```diff
@@ -63,10 +63,6 @@ function HeroComponent({ className, changeUI, browsePage }) {
             image={globeExplore}
             handleClick={() => {
               browsePage({ type: DATA });
-              changeUI({
```

Comment:
why are we removing this? Its part of the onboarding link

Reply:
We have remove it from mobile components as we are not using onboarding for these devices